### PR TITLE
SQL syntax highlighting

### PR DESCRIPTION
**Pull request by msftristew**
We need to add kernel.js files for the kernels. They have the same content.

### CODE REVIEW

**Comment by msftristew** [comment]
Closes #53

File: remotespark/pysparkkernel/kernel.js [hunk 1]
```diff
@@ -0,0 +1 @@
+IPython.CodeCell.config_defaults.highlight_modes['magic_text/x-sql'] = {'reg':[/^%%sql/, /^%%hive/]};
```
Comment (rgbkrk) [inline remotespark/pysparkkernel/kernel.js]: Have you figured out a way to use the hive dialect here?

Comment (aggFTW) [inline remotespark/pysparkkernel/kernel.js]: You mean syntax highlight HQL? We just use the SQL lexer.
Sorry if my answer isn't clear, but I'm not sure I understood. Use it for what?

Comment (rgbkrk) [inline remotespark/pysparkkernel/kernel.js]: The default here is the ANSI SQL dialect, [there's also a hive mime mode that supports additional keywords with the Hive SQL dialect](https://github.com/codemirror/CodeMirror/blob/c4387d6073b15ccf0f32773eb71a54f3b694f2f0/mode/sql/sql.js#L394-L402).

Comment (aggFTW) [inline remotespark/pysparkkernel/kernel.js]: Cool! Did you want to change it to Hive? We'll get to that later (no one's asked for it yet), but nice to know it exists!

**Pull request merged**